setup.py

```diff
@@ -10,7 +10,7 @@
     'django-mptt>=0.6,<1.0',  # the exact version depends on Django
     'django_polymorphic>=0.7,<2.1',
     'easy-thumbnails>=2,<3.0',
-    'Unidecode>=0.04,<1.1',
+    'Unidecode>=0.04,<=1.1.1',
```

Comment:
wouldn't `Unidecode>=0.04,<1.2` make more sense?

Reply:
Yes, I hurriedly made the change for my own repo & my own requirements. <1.2 is better.